packages/web/src/app/clubs/page.tsx

```diff
@@ -1,11 +1,170 @@
 "use client";
 
+import SectionTitle from "@sparcs-clubs/web/common/components/SectionTitle";
 import { UseClientProvider } from "@sparcs-clubs/web/common/providers/UseClientProvider";
+import styled from "styled-components";
+import Card from "@sparcs-clubs/web/common/components/Card";
+import Tag from "@sparcs-clubs/web/common/components/Tag";
+import { useState } from "react";
+import clubsData from "./data/clubs";
 
-const Clubs = () => (
-  <main>
-    <UseClientProvider>{/* TODO */}</UseClientProvider>
-  </main>
-);
+interface ClubsData {
+  id: number;
+  clubName: string;
+  characteristicKr: string;
+  clubType: string;
+  clubPresident: string;
+  advisor: string | null;
+  totalMembers: number;
+}
+interface CardGridProps {
+  clubsDataList: ClubsData[];
+}
+
+const CardGridInner = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 16px;
+  padding-left: 24px;
+`;
+
+const Section = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`;
+
+const ClubCategoryTitle = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  justify-content: space-between;
+`;
+
+const MoreInfo = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration-line: underline;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+`;
+
+const CardInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+  flex: 1 0 0;
+`;
+
+const ClubName = styled.div`
+  flex: 1 0 0;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+`;
+
+const ClubCardHead = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  align-self: stretch;
+`;
+
+const ClubMembers = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 4px;
+`;
+
+const ClubMembersInner = styled.div`
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 16px;
+`;
+
+const IconPlaceHolder = styled.div`
+  width: 16px;
+  height: 16px;
+  background-color: #000;
+`;
+
+const ClubDetail = styled.div`
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+`;
+
+const getTagColor = (clubType: string) => {
+  switch (clubType) {
+    case "상임동아리":
+      return "GREEN";
+    case "가동아리":
+      return "ORANGE";
+    default:
+      return "BLUE";
+  }
+};
+
+// eslint-disable-next-line react/prop-types
+const CardGrid: React.FC<CardGridProps> = ({ clubsDataList = [] }) => {
+  const sortedClubs = clubsDataList.sort((a, b) => {
+    if (a.clubType === "상임동아리" && b.clubType !== "상임동아리") {
+      return -1;
+    }
+    if (a.clubType !== "상임동아리" && b.clubType === "상임동아리") {
+      return 1;
+    }
+    return 0;
+  });
```

Comment:
```suggestion
  const clubTypeOrder = ["상임동아리", "정동아리", "가동아리"];
  const sortedClubs = clubsDataList.sort(
    (a, b) =>
      clubTypeOrder.indexOf(a.clubType) - clubTypeOrder.indexOf(b.clubType),
  );
```
동아리의 종류가 3개보다 많아지면???! 해당 방법의 정렬로는 이해하기 어려울 수도 있을 것 같아요!
물론 더 좋은 방법이 있을 수 있지만.. 당장 생각나는 방법 한 번 써봤어요
이런 식으로 하면 무엇이 우선으로 보여야 하는지 더 직관적이다는 장점도 있을 것 같아요!!

Reply:
오 order를 리스트로 만들어서 쓰는 건 생각 못했는데 좋은 것 같아요

Reply:
참고로 hama님이나 ava님 코드 보면 enum으로 정의해서 사용하는 방법도 있습니다!!